<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="59.1750" tests="5" failures="2">
  <testsuite name="Root Suite" timestamp="2021-12-06T14:17:58" tests="0" file="cypress/integration/End2EndTest.spec.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Test Case - 1" timestamp="2021-12-06T14:17:59" tests="1" time="9.5580" failures="0">
    <testcase name="Test Case - 1 Testing Menu" time="9.6520" classname="Testing Menu">
    </testcase>
  </testsuite>
  <testsuite name="Test Case - 2" timestamp="2021-12-06T14:18:08" tests="1" time="38.0090" failures="0">
    <testcase name="Test Case - 2 Testing Body" time="37.9040" classname="Testing Body">
    </testcase>
  </testsuite>
  <testsuite name="Test Case - 3, Mocking Network Response " timestamp="2021-12-06T14:18:46" tests="1" time="0.9270" failures="0">
    <testcase name="Test Case - 3, Mocking Network Response  Mocking Network Response" time="0.9260" classname="Mocking Network Response">
    </testcase>
  </testsuite>
  <testsuite name="Testing API Endpoints" timestamp="2021-12-06T14:18:47" tests="1" time="5.2640" failures="1">
    <testcase name="Testing API Endpoints Test Get Request" time="0.0000" classname="Test Get Request">
      <failure message="Timed out retrying after 4000ms: expected [ Array(7) ] to have a length of 6 but got 7" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: expected [ Array(7) ] to have a length of 6 but got 7
    at frontEnd.restApiTesting (http://localhost/__cypress/tests?p=cypress/integration/End2EndTest.spec.js:7590:51)
    at Context.eval (http://localhost/__cypress/tests?p=cypress/integration/End2EndTest.spec.js:7345:34)]]></failure>
    </testcase>
  </testsuite>
  <testsuite name="Backend Testing" timestamp="2021-12-06T14:18:52" tests="1" time="5.2300" failures="1">
    <testcase name="Backend Testing Get Request" time="0.0000" classname="Get Request">
      <failure message="Timed out retrying after 4000ms: expected [ Array(8) ] to have a length of 7 but got 8" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: expected [ Array(8) ] to have a length of 7 but got 8
    at frontEnd.backEndTesting (http://localhost/__cypress/tests?p=cypress/integration/End2EndTest.spec.js:7617:51)
    at Context.eval (http://localhost/__cypress/tests?p=cypress/integration/End2EndTest.spec.js:7356:34)]]></failure>
    </testcase>
  </testsuite>
</testsuites>